src/test/java/org/openmrs/contrib/qaframework/page/RegistrationSummaryPage.java

```diff
@@ -14,16 +14,48 @@
 
 public class RegistrationSummaryPage extends Page {
 
+	private static final By EDIT_DEMOGRAPHICS_LINK = By.cssSelector("#edit-patient-demographics a");
+	private static final By SHOW_CONTACT_INFO_LINK = By.id("patient-header-contactInfo");
+	private static final By EDIT_CONTACT_INFO_LINK = By.id("contact-info-inline-edit");
 	private static final By EDIT_RELATIONSHIP_LINK = By.id("relationships-info-edit-link");
+	private static final By PATIENT_GIVENNAME = By.cssSelector("#content div span.PersonName-givenName");
+	private static final By PATIENT_FAMILYNAME = By.cssSelector(".patient-header .demographics .name .PersonName-familyName");
+	private static final By TELEPHONE_NUMBER_TEXT = By.cssSelector("#coreapps-telephoneNumber");
```

Comment:
@kdaud  done.

Reply:
@Irenyak1 are they texts or fields?

Reply:
@kdaud they are texts,  So I am basically picking texts so that I can assert their presence on the page after updating the patient demographics and contact information.

Reply:
I guess these are fields where patient names and contacts are field, isn't it so?

Reply:
@Irenyak1 did you see the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/153#discussion_r749276067)?

Reply:
@kdaud no they are not fields but rather just spans displaying what has been captured.  They are just displaying text captured after the editing is done, they are not fields, they have no provision for filling in data.